NuclearData/ceNeutronData/ceNeutronDatabase_inter.f90

```diff
@@ -196,7 +197,31 @@ subroutine updateMicroXSs(self, E, nucIdx, rand)
     end subroutine updateMicroXSs
 
     !!
-    !! Subroutine to get the elastic scattering majorant cross section in a nuclide
+    !! Subroutine to retrieve the nuclide total majorant cross section
+    !! over a calculated energy range (needed for TMS) and update the nuclide
```

Comment:
> It refers to the span of relative energies a nuclide can see given a material temperature, right?

Yes, exactly. I tried to leave the comment generic because potentially you can use this function with an arbitrary energy range if you wanted too. It doesn't have to be specific to TMS. I can be more specific in the comments though